src/pages/home/sidebar/GlobalNavigation/GlobalNavigation.js

```diff
@@ -0,0 +1,50 @@
+import React, {useMemo} from 'react';
+import {View} from 'react-native';
+import PropTypes from 'prop-types';
+import PressableAvatarWithIndicator from '../PressableAvatarWithIndicator';
+import styles from '../../../../styles/styles';
+import * as Expensicons from '../../../../components/Icon/Expensicons';
+import GlobalNavigationMenuItemList from './GlobalNavigationMenuItemList';
+import CONST from '../../../../CONST';
+import Navigation from '../../../../libs/Navigation/Navigation';
+import ROUTES from '../../../../ROUTES';
+
+const propTypes = {
+    isCreateMenuOpen: PropTypes.bool,
+};
+
+const defaultProps = {
+    isCreateMenuOpen: false,
+};
+
+function GlobalNavigation({isCreateMenuOpen}) {
```

Comment:
Linter will throw an error 
`propType "isCreateMenuOpen" is not required, but has no corresponding defaultProps declaration`

But this may be a good to modify the rules for eslint

Reply:
well I think we removed `isCreateMenuOpen` so we don't have to worry about it here